src/pages/index.js

```diff
@@ -218,23 +223,30 @@ function handleAvatarSubmit(evt) {
 }
 
 function handlePostCard(evt) {
-  evt.preventDefault();
-  const postCardSubmitBtn = evt.submitter;
-  setButtonText(postCardSubmitBtn, true);
-
+  renderLoading(true, cardAddSubmitButton);
   api
-    .postCard({ name: newPostCaption.value, link: newPostImageLink.value })
-    .then((data) => {
-      const newCard = getCardElement(data);
-
-      cardsList.prepend(newCard);
-
-      closeModal(addCardModal);
-      evt.target.reset();
+    .postCard(data)
```

Comment:
* When I add a new card, the page **reloads** 


`  evt.preventDefault();` is needed here


```js
function handlePostCard(evt) {
  evt.preventDefault();
  evt.preventDefault();
  const cardAddSubmitButton = evt.submitter;
  setButtonText(cardAddSubmitButton, true);

```